### PR TITLE
Update start page (based on Jay's edits)

### PR DESCRIPTION
**Pull request by abbycross**
Continuation of https://github.ibm.com/IBM-Q-Software/iqp-channel-docs/pull/794

### CODE REVIEW

**Review by blakejohnson — APPROVED** [review]

**Comment by abbycross** [comment]
@jaygambetta to make it easier, see the preview of the page here: https://qiskit-docs-preview-pr-73.1799mxdls7qz.us-south.codeengine.appdomain.cloud/start

File: docs/start/index.mdx [hunk 1]
```diff
@@ -8,21 +8,35 @@ in_page_toc_max_heading_level: 2
...
+
+
+
+- **Scalable and flexible quantum software**. 
+
+  IBM Quantum is committed to open-source innovation. Preferred by 81% of quantum programmers[^2], the Qiskit software development kit works with eight different hardware manufacturers, making it the lingua franca of quantum computing, and its circuit optimization is the most advanced (seven times faster utility-scale circuit transpilation than competing frameworks). Build quantum circuits and operators in Qiskit, the transpile and run on quantum systems using primitives, a simplified interface for circuit execution. When you pair Qiskit with Qiskit Runtime, you'll benefit from seamless circuit execution powered by advanced runtime compilation, error suppression, and error mitigation techniques.
```
Comment (javabster) [inline docs/start/index.mdx:24]:
```suggestion
  IBM Quantum is committed to open-source innovation. Preferred by 81% of quantum programmers[^2], the Qiskit software development kit works with eight different hardware manufacturers, making it the lingua franca of quantum computing, and its circuit optimization is the most advanced (seven times faster utility-scale circuit transpilation than competing frameworks). Build quantum circuits and operators in Qiskit, then transpile and run on quantum systems using primitives, a simplified interface for circuit execution. When you pair Qiskit with Qiskit Runtime, you'll benefit from seamless circuit execution powered by advanced runtime compilation, error suppression, and error mitigation techniques.  
```

**Review by javabster — COMMENTED** [review]
One concern I have with this intro is that it is quite lengthy. imho the key information we want to convey to people are the steps of the typical users workflow and how the documentation is split up into those sections: start -> build -> (transpile TBC) -> verify -> run

Right now when I view the preview I feel like I'm confronted with a wall of text and I have to scroll before I even reach the explanation of the workflow steps. In addition the language used in the intro feels maybe a smidge too close to marketing comms than technical documentaion for my taste. I'm not saying we shouldn't say anything about how great our stuff is, but we could probably cut it down a little

**Comment by javabster** [comment]
This would be my suggestion for dramatically cutting down the intro of this section while still keeping the utility tone:

> The quantum community has explored quantum computing on the cloud since 2016.  Now the community has a new challenge: *demonstrate useful quantum computation*. 
> 
> Through IBM Quantum, you can leverage performant and utility-scale (>100-qubit) quantum systems for your work, accessible using scalable and flexible quantum software. Whether you try out a no-cost plan or run your programs through a pay-as-you-go or premium plan, you can start doing useful work with quantum computing.
> 
>  The sections at the core of our documentation are based on the stages of a typical quantum user’s journey. One where 
> you build quantum circuits and operators in Qiskit, then transpile and run on quantum systems using Qiskit Runtime primitives, a simplified interface for circuit execution. By pairing Qiskit with Qiskit Runtime, you benefit from seamless circuit execution powered by advanced runtime compilation, error suppression, and error mitigation techniques. 
> 
> If you’re just getting started with IBM Quantum, you are in the right place. Use the topics in this section to get ready, then proceed to [Build](link) to create your first quantum circuit. Keep working through the sections in order, following the tabs at the top of this page.
> etc. etc.

I liked some of the stats/references about the reliability of the hardware, but I think that shoutout could be moved to the intro of the run section. I also like the open source stats too, but think it could be better highlighted in a dedicated "What is Qiskit" page

**Review by javabster — APPROVED** [review]
LGTM 🚀

**Pull request merged**